components/typeahead/typeahead.directive.ts

```diff
@@ -220,6 +220,16 @@ export class Typeahead implements OnInit {
     }
   }
 
+  @HostListener('keydown', ['$event'])
+  onKeydown(e:KeyboardEvent) {
+    // When typeahead container is visible, prevent submitting the form
+    if (this.container) {
```

Comment:
please reduce nesting
`if (this.container && e.keyCode === 13)`
should be enough